azkaban-db/src/main/sql/create.containerization-tables-all.sql

```diff
@@ -72,10 +72,10 @@ CREATE TABLE IF NOT EXISTS image_rampup_plan (
 --  One option is to move each table create scripts to separate file. But all the  containerization
 --  tables are placed in this file so that it easier to manage.
 
-CREATE INDEX image_rampup_plan_type_id
+CREATE INDEX IF NOT EXISTS image_rampup_plan_type_id
```

Comment:
`IF NOT EXISTS` isn't valid with `CREATE INDEX`. Please refer to line 70 above for a possible solution.

Reply:
I tried to solve this issue with a stored procedure, but ran into issues due to how we have DatabaseOperator implemented, if I remember correctly.

Reply:
@ypadron-in you're right. It still breaks web server in the debug mode. I tried to use a stored procedure instead, but received syntax error (which worked on mysql workbench...). I'll discard this commit.